backend/seed.py

```
@@ -161,6 +162,38 @@ def load_hints():
     logger.info("Hints loaded successfully.")
 
 
+def load_code_checks():
+    # Load modules with BONUS_CHALLENGE and CHALLENGE types and create a mapping of module titles to modules
+    modules = (
+        db.session.query(Module)
+        .filter(
+            Module.module_type.in_([ModuleType.BONUS_CHALLENGE, ModuleType.CHALLENGE])
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)